File: Java/KodeProgram/Handphone.java
// Handphone.java
// Berisi definisi class Handphone, turunan dari Elektronik.
class Handphone extends Elektronik {
    private String os;
    private int camera;
    private int battery;

    public Handphone(int id, String nama, double harga,
                     String brand, int garansi, int stok,
                     String os, int camera, int battery) {
        super(id, nama, harga, brand, garansi, stok);
        this.os = os;
        this.camera = camera;
        this.battery = battery;
    }

    @Override
    public String getKategori() { return "Handphone"; }

    @Override
    public String getDetail() {
        return "OS:" + os + ", Camera:" + camera + "MP, " + battery + "mAh";
    }
}

File: Python/KodeProgram/Handphone.py
# Handphone.py
# Turunan Level 3: Handphone
from Elektronik import Elektronik

class Handphone(Elektronik):
    def __init__(self, id, nama, harga, brand, garansi, stok, os, camera, battery):
        super().__init__(id, nama, harga, brand, garansi, stok)
        self.__os = os
        self.__camera = camera
        self.__battery = battery

    def get_kategori(self):
        return "Handphone"

    def get_detail(self):
        return f"OS:{self.__os}, Camera:{self.__camera}MP, {self.__battery}mAh"